CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+Fork [moban](https://github.com/moremoban/moban) by clicking "Fork".
+
+## Using virtualenv and Remote Configuration
+1. `pip3 install virtualenv`
+2. Move to the location where you want to setup moban and type `mkdir moban` in the terminal
+3. `cd moban`
+3. Type `virtualenv venv`
+4. `source venv/bin/activate`
+5. `git clone https://github.com/YOUR_USERNAME/moban.git`
+6. `cd moban`
+7. `pip3 install -e .`
+7. `git remote add origin https://github.com/YOUR_USERNAME/moban.git`
```

Comment:
when we clone a repo from our fork then the origin is already set so this would most probably give an error.

Reply:
yupp